src/annotator/components/NotebookModal.js

```diff
@@ -78,24 +78,22 @@ export default function NotebookModal({ eventBus, config }) {
   }, [isHidden]);
 
   useEffect(() => {
-    emitter.current = eventBus.createEmitter();
-    emitter.current.subscribe(
-      'openNotebook',
-      (/** @type {string} */ groupId) => {
-        setIsHidden(false);
-        setIframeKey(iframeKey => iframeKey + 1);
-        setGroupId(groupId);
-      }
-    );
+    const newEmitter = eventBus.createEmitter();
```

Comment:
These changes fix two complaints about `emitter.current` being possibly null. In the `subscribe` call we know that's not the case. In the cleanup callback that `useEffect` returns however there is the possibility that `emitter.current` gets re-assigned before the callback is run. The change here avoids that hazard.